UltiSnips/django.snippets

```diff
@@ -1,238 +1,367 @@
 priority -50
 
-# Generic Tags
-snippet %
-{% ${1} %}${2}
+# This files will define django snippets from sublime text djaneiro
+
+###########################################################################
+#                             FORMS SNIPPETS                              #
+###########################################################################
+snippet form "Form"
```

Comment:
A brand new feauture, so it is not yet used in vim-snippets anywhere. There are examples in the tests here:

https://github.com/SirVer/ultisnips/blob/master/test/test_ContextSnippets.py#L15

@seletskiy do you have real world examples of context snippets?